_rules/first-focusable-element-is-link-to-main-8a213c.md

```diff
@@ -0,0 +1,687 @@
+---
+id: 8a213c
+name: First focusable element is link to main area of content
+rule_type: atomic
+description: |
+  This rule checks that the first focusable element is a link to non-repeated content in the page
+accessibility_requirements:
+  wcag-technique:G1: # Adding a link at the top of each page that goes directly to the main content area
+    forConformance: false
+    failed: not satisfied
+    passed: satisfied
+    inapplicable: further testing needed
+input_aspects:
+  - Accessibility Tree
+  - CSS styling
+  - DOM Tree
+  - Language
+acknowledgments:
+  authors:
+    - Christina Adams
+    - Jean-Yves Moyen
+  assets:
+    - _The Three Kingdoms_ by Luo Guanzhong, translation by Yu Sumei (Tuttle publishing, 2014, ISBN 9780804843935)
+---
+
+## Applicability
+
+This rule applies to any [HTML web page][].
+
+## Expectation 1
+
+Each test target has a non-empty [sequential focus navigation order][sequential focus navigation].
+
+## Expectation 2
+
+For each test target, all the following are true for the first element in its [sequential focus navigation order][sequential focus navigation]:
+
+- the focusable element is [keyboard actionable][]; and
+- the focusable element is [included in the accessibility tree][]; and
+- the focusable element is a [semantic link][]; and
+- when the focusable element is [activated][], focus moves [at the end][] of the first [block of repeated content][] (in the [flat tree][]) with [perceivable content][] [at its end][at the end] which is not part of any [block of repeated content][]; and
```

Comment:
This is pretty confusing, can you rephrase this in some way? I'm not sure I understand what this is saying.